website/src/components/canvas/Canvas.jsx

```diff
@@ -194,6 +199,26 @@ class Canvas extends React.Component {
     this.hasInitialized = true;
   }
 
+
+  /**
+  * Key press handler
+  * @param {Object} event is the event listener for the window
+  * @returns {void}
+  */
+  handleKeyPress(event) {
+    if(this.props.targetedId){
+      if(event.code === "Delete"){
+        this.props.dispatch(ElementActions.removeElementAndPersist(
+          this.props.targetedId, this.props.params.canvasId));
+      }
+      else if(event.ctrlKey === true && event.code === "KeyV") {
```

Comment:
I would change this line to just if rather than else-if, since you have no terminating else.
I can do it in my PR tho, so good job!